tests/functional/oauth_webchannel.js

```diff
@@ -2,6 +2,9 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+// TODO: remove once https://github.com/eslint/eslint/issues/1801 gets fixed
+/* eslint indent: 0 */
+
```

Comment:
So, we will use both TODO and HACK for the same stuff? or do you want me to change all the TODOs to HACKs?